it/v3/highIncomeChildBenefitCharge/delete/DeleteHighIncomeChildBenefitChargeControllerISpec.scala

```diff
@@ -0,0 +1,151 @@
+          ("AA1123A", "2025-26", BAD_REQUEST, NinoFormatError),
+          ("AA123456A", "20477",  BAD_REQUEST, TaxYearFormatError),
+          ("AA123456A", "2025-27",  BAD_REQUEST, RuleTaxYearRangeInvalidError),
+          ("AA123456A", "2024-25",  BAD_REQUEST, RuleTaxYearNotSupportedError)
+        )
+        input.foreach(args => (validationErrorTest _).tupled(args))
+      }
+
+      "downstream service error" when {
+        def serviceErrorTest(downstreamStatus: Int, downstreamCode: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
+          s"downstream returns an $downstreamCode error and status $downstreamStatus" in new Test {
+
+            override def setupStubs(): StubMapping = {
+              AuditStub.audit()
+              AuthStub.authorised()
+              MtdIdLookupStub.ninoFound(nino)
+              DownstreamStub.onError(DownstreamStub.DELETE, downstreamUri, downstreamStatus, errorBody(downstreamCode))
+            }
+
+            val response: WSResponse = await(request().delete())
+            response.status shouldBe expectedStatus
+            response.json shouldBe Json.toJson(expectedBody)
+            response.header("Content-Type") shouldBe Some("application/json")
+          }
+        }
+
+        val errors = List(
+          (BAD_REQUEST, "INVALID_TAXABLE_ENTITY_ID", BAD_REQUEST, NinoFormatError),
+          (BAD_REQUEST, "INVALID_TAX_YEAR", BAD_REQUEST, TaxYearFormatError),
+          (BAD_REQUEST, "INVALID_CORRELATIONID", INTERNAL_SERVER_ERROR, InternalError),
+          (NOT_FOUND, "NOT_FOUND", NOT_FOUND, NotFoundError),
+          (UNPROCESSABLE_ENTITY, "TAX_YEAR_NOT_SUPPORTED", BAD_REQUEST, RuleTaxYearNotSupportedError),
+          (UNPROCESSABLE_ENTITY, "OUTSIDE_AMENDMENT_WINDOW", BAD_REQUEST, RuleOutsideAmendmentWindow),
+          (INTERNAL_SERVER_ERROR, "SERVER_ERROR", INTERNAL_SERVER_ERROR, InternalError),
+          (SERVICE_UNAVAILABLE, "SERVICE_UNAVAILABLE", INTERNAL_SERVER_ERROR, InternalError)
+        )
+
+        errors.foreach(args => (serviceErrorTest _).tupled(args))
+      }
+
+    }
+  }
+
+  private trait Test {
+
+    val nino = "AA123456A"
+
+    def taxYear: String           = "2025-26"
+    def downstreamTaxYear: String = "25-26"
+
+    def downstreamUri: String = s"/itsa/income-tax/v1/$downstreamTaxYear/high-income-child-benefit/charges/$nino"
+
+    def mtdUri: String = s"/high-income-child-benefit/$nino/$taxYear"
+
+    def setupStubs(): StubMapping
+
+    def request(): WSRequest = {
+      AuthStub.resetAll()
+      setupStubs()
+      buildRequest(mtdUri)
+        .withHttpHeaders(
+          (ACCEPT, s"application/vnd.hmrc.3.0+json"),
+          (AUTHORIZATION, "Bearer 123") // some bearer token
+        )
+    }
+
+    def errorBody(code: String): String =
+      s"""{
+         |  "failures": [
+         |    {
+         |      "code": "$code",
+         |      "reason": "Downstream message."
+         |    }
+         |  ]
+         |}
+    """.stripMargin
```

Comment:
This doesn't match the downstream error format refer to this here
https://github.com/hmrc/individuals-charges-api/blob/7aaada1a1d8eb8b432b6b1bf6f2b65013067322b/it/v3/highIncomeChildBenefitCharge/RetrieveHighIncomeChildBenefitISpec.scala#L100-L113